ocp_utilities/operators.py

```diff
@@ -339,14 +352,40 @@ def _icsp(_repository_digest_mirrors):
         admin_client=admin_client,
     )
 
-    catalog_source = CatalogSource(
+    iib_catalog_source = create_catalog_source_from_image(
+        admin_client=admin_client,
         name=name,
         namespace=operator_market_namespace,
-        display_name=name,
         image=_iib_index_image,
+    )
+    return iib_catalog_source
+
+
+def create_catalog_source_from_image(
+    name, namespace, image, source_type=None, update_strategy_registry_poll_interval=None, admin_client=None
+):
+    """
+    Create CatalogSource for given image
```

Comment:
Missing empty line between sections. (after first description line and before Returns)

Reply:
Added